sdks/python/apache_beam/examples/complete/game/setup.py

```diff
@@ -0,0 +1,35 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Setup.py module for the workflow's worker utilities.
```

Comment:
<!--thread_id:cc_128826194_t; commit:95d457e2ad40bb613cb8abda68903ad026f0cccd; resolved:0-->
<!--section:context-quote-->
> **davidcavazos** wrote:
> I wasn't able to run on Dataflow without this, but it probably was because the Dataflow backend wasn't using the my modified ``apache_beam`` package which included this new module.
> 
> As a side note, for importing single files, it looks like too much of a hassle to create the ``setup.py``. Would it be feasible to let the runner detect if there is a local file matching an imported module name, package that into a module using setuptools, and send it automatically, instead of writing the whole setuptools instructions manually? That would also get rid of the inconsistency between importing in the DirectRunner and DataflowRunner.

<!--section:body-->
Any pipeline, that requires multiple files require a setup.py today. Having it all in `apache_beam` would work, but note that they would not have worked if they were not part of the core sdk like. For example, this would fail if examples were living in a different repo.

I think it is good practice to organize project with a setup.py, to avoid getting complicated dependencies. An automatic way of doing that might reduce the short term hassle but in the long term would probably cause more disorganization in the user code.